contracts/OpenSTHelper.sol

```diff
@@ -0,0 +1,99 @@
+pragma solidity ^0.4.23;
+
+
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import "./BytesLib.sol";
+import "./MerklePatriciaProof.sol";
+import "./RLPEncode.sol";
+
+
+/**
+  *	@title OpenSTHelper
+  */
+// Please note that this library exists for 0.9.3 and going forward will be merged with Utils.sol.
+library OpenSTHelper {
+
+  /**
+    *	@notice Convert bytes32 to bytes
+    *
+    *	@param a bytes32 value
+    *
+    *	@return bytes value
+    */
+  function bytes32ToBytes(bytes32 a) internal pure returns (bytes) {
+    bytes memory res = new bytes(32);
+    assembly {
+      mstore(add(32,res), a)
+    }
+    return res;
+  }
+
+  /**
+    *	@notice Get the storage path of the variable
+    *
+    *	@param _index Index of variable
+    *	@param _key Key of variable incase of mapping
+    *
+    *	@return bytes32 Storage path of the variable
+    */
+  function storagePath(
```

Comment:
Name is confusing.

1. Do we name in other part of codebase (or we took it from community) the name "path". For me it's encoded index for storage variable.
2. If the answer is yes, then, let's have storageVariablePath().

Reply:
changed to `storageVariablePath()`